### ISSUE TITLE
pass other empirica config options as a flag

### ISSUE THREAD

**Issue by JamesPHoughton** [empiricaly/empirica#382]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Is your feature request related to a problem?

I want to set the admin pw from an environment variable, rather than committing it with my code

### Describe the solution you'd like

Currently, I can use the `--tajriba.store.file` flag to specify a new location for the `tajriba.json` file. It would be nice to be able to overwrite any of the options in `.empirica/empirica.toml` file when launching empirica, using a flag argument. 

Something like:
```
empirica --config username=myAdminName --config password=myAdminPw
```

### Describe alternatives you've considered

I've tried to edit empirica.toml prior to launching the server (using sed) but I'm not particularly good at it.

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by npaton** [comment]
Most options have a command line option that corresponds the value in empirica.toml. I think users is actually the only one that doesn't since it's an array of objects and that's hard to represent with flags. I will see what we can do, but I'm not sure.

**Comment by npaton** [comment]
Fixed in 1.6.1. See [release notes](https://github.com/empiricaly/empirica/releases/tag/%40empirica%2Fcore%401.6.1) for details.
